gui/src/app/SPAnalysis/SPAnalysisContextProvider.tsx

```diff
@@ -34,18 +48,31 @@ const SPAnalysisContextProvider: FunctionComponent<PropsWithChildren<SPAnalysisC
             window.removeEventListener('beforeunload', handleBeforeUnload);
         };
     }, [data])
+    ////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Wondering if we still need to block out this section of the code--seems like we might be happy sticking with this behavior?